Dockerfile

```diff
@@ -109,18 +109,16 @@ RUN patch -d / -p0 < /tmp/ssh_q.patch
 COPY hosted-ce/overrides/bosco_cluster_xtrace.patch /tmp
 RUN patch -d / -p0 < /tmp/bosco_cluster_xtrace.patch
 
-# HACK: Don't copy over the SSH pub key to the remote side. We set
-# this up with the site out of band.
+# FIXME: Remove this check after a successful build
+# Don't copy the SSH key (HTCONDOR-270)
 COPY hosted-ce/overrides/HTCONDOR-270.skip-key-copy.patch /tmp
```

Comment:
Shouldn't you delete this line, since you're deleting the file?

Reply:
Oops yeah, missed it.